PHP/src/Aseo/Api/V3/Serps/SerpsApiClient.php

```diff
@@ -0,0 +1,132 @@
+<?php
+namespace Aseo\Api\V3\Serps;
+
+/**
+* Analytics SEO PHP CLient
+*
+* Serach Engine results page CLient
+
+* @version 3
+*
+* @author Nuno Franco da Costa <nuno@francodacosta.com>
+* @copyright 2014 Analytics SEO ltd
+* @license http://www.opensource.org/licenses/mit-license.php MIT
+* @link https://github.com/analyticsseo/api-client/
+*/
+
+use Aseo\Api\Auth\KeyAuth;
+use \Guzzle\Http\Client as GuzzleClient;
+
+class SerpsApiClient
+{
+    /**
+     * HTTP transport
+     * @var \Guzzle\Http\Client
+     */
+    private $transport;
+
+    /**
+     * AUthentication Mechanism
+     * @var KeyAuth
+     */
+    private $auth;
+
+    public function __construct(\Guzzle\Http\Client $transport, KeyAuth $auth)
+    {
+        $this->setTransport($transport);
+        $this->setAuth($auth);
+    }
+
+    /**
+     * Get the value of HTTP transport
+     *
+     * @return \Guzzle\Http\Client
+     */
+    public function getTransport()
+    {
+        return $this->transport;
+    }
+
+    /**
+     * Set the value of HTTP transport
+     *
+     * @param \Guzzle\Http\Client transport
+     *
+     * @return self
+     */
+    public function setTransport(\Guzzle\Http\Client $transport)
```

Comment:
This is already initialised in the constructor, I would make this private or protected.